### ISSUE TITLE
Hide runtime stats from Presto CLI in debug mode

### ISSUE THREAD

**Issue by tdcmeehan** [prestodb/presto#22041]
## Your Environment
Not relevant

## Expected Behavior
When I used the CLI, I see a large amount of runtime stats when run in debug mode when the query is finished, which impedes my ability to see query results:

```
S0-driverCountPerTask: sum=1 count=1 min=1 max=1
S0-taskBlockedTimeNanos: sum=0:01 count=1 min=0:01 max=0:01
S0-taskElapsedTimeNanos: sum=0:02 count=1 min=0:02 max=0:02
S0-taskQueuedTimeNanos: sum=0:01 count=1 min=0:01 max=0:01
S0-taskScheduledTimeNanos: sum=70ms count=1 min=70ms max=70ms
S1-driverCountPerTask: sum=20 count=4 min=5 max=5
S1-taskBlockedTimeNanos: sum=0:08 count=4 min=0:02 max=0:02
S1-taskElapsedTimeNanos: sum=0:05 count=4 min=0:01 max=0:01
S1-taskQueuedTimeNanos: sum=0:02 count=4 min=380ms max=396ms
S1-taskScheduledTimeNanos: sum=0:03 count=4 min=0:01 max=0:01
S2-driverCountPerTask: sum=1 count=1 min=1 max=1
S2-getSplitsTimeNanos: sum=0ms count=1 min=0ms max=0ms
S2-taskElapsedTimeNanos: sum=376ms count=1 min=376ms max=376ms
S2-taskQueuedTimeNanos: sum=58ms count=1 min=58ms max=58ms
S2-taskScheduledTimeNanos: sum=35ms count=1 min=35ms max=35ms
fragmentPlanTimeNanos: sum=67ms count=1 min=67ms max=67ms
getCanonicalInfoTimeNanos: sum=0ms count=1 min=0ms max=0ms
getColumnHandleTimeNanos: sum=1ms count=1 min=1ms max=1ms
getColumnMetadataTimeNanos: sum=0ms count=1 min=0ms max=0ms
getLayoutTimeNanos: sum=13ms count=5 min=0ms max=7ms
getMaterializedViewTimeNanos: sum=0ms count=1 min=0ms max=0ms
getTableHandleTimeNanos: sum=0ms count=1 min=0ms max=0ms
getViewTimeNanos: sum=0ms count=1 min=0ms max=0ms
logicalPlannerTimeNanos: sum=49ms count=1 min=49ms max=49ms
optimizerTimeNanos: sum=133ms count=1 min=133ms max=133ms
[Latency: client-side: 0:03, server-side: 0:03] [8 rows, 166B] [2 rows/s, 50B/s]
```

## Current Behavior
Runtime stats are returned with the query results for every query in debug mode

## Possible Solution
This should be an opt-in feature, as it is only relevant for Presto developers and highly technical users.

## Steps to Reproduce
Run any query in the CLI in debug mode

## Screenshots (if appropriate)

See above example.

## Context
Runtime stats were recently hidden from the UI.  They should be hidden here as well.

**Comment by efrancisworks** [comment]
Hi, I would like to work on this issue.

**Comment by tdcmeehan** [comment]
Fixed by #22436
